benchmark/benchmark-tidb-using-sysbench.md

```diff
@@ -109,10 +111,10 @@ Restart MySQL client and execute the following SQL statement to create a databas
 create database sbtest;
 ```
 
-Adjust the order in which Sysbench scripts create indexes. Sysbench imports data in the order of "Build Table -> Insert Data -> Create Index", which takes more time for TiDB to import data. Users can adjust the order to speed up the import of data. Suppose that you use the Sysbench version [1.0.14](https://github.com/akopytov/sysbench/tree/1.0.14). You can adjust the order in either of the following two ways:
+Adjust the order in which Sysbench scripts create indexes. Sysbench imports data in the order of "Build Table -> Insert Data -> Create Index", which takes more time for TiDB to import data. Users can adjust the order to speed up the import of data. Suppose that you use the Sysbench version [1.0.20](https://github.com/akopytov/sysbench/tree/1.0.20). You can adjust the order in either of the following two ways:
 
 - Download the modified [oltp_common.lua](https://raw.githubusercontent.com/pingcap/tidb-bench/master/sysbench/sysbench-patch/oltp_common.lua) file for TiDB and overwrite the `/usr/share/sysbench/oltp_common.lua` file with it.
-- In `/usr/share/sysbench/oltp_common.lua`, move the lines [235](https://github.com/akopytov/sysbench/blob/1.0.14/src/lua/oltp_common.lua#L235)-[240](https://github.com/akopytov/sysbench/blob/1.0.14/src/lua/oltp_common.lua#L240) to be right behind the line 198.
+- In `/usr/share/sysbench/oltp_common.lua`, move the lines [235-240](https://github.com/akopytov/sysbench/blob/1.0.20/src/lua/oltp_common.lua#L235-L240) to be right behind the line 198.
```

Comment:
https://github.com/akopytov/sysbench/blob/ebf1c90da05dea94648165e4f149abc20c979557/src/lua/oltp_common.lua#L235-L240

```
$ sysbench oltp_read_only help
sysbench 1.1.0-df89d34 (using bundled LuaJIT 2.1.0-beta3)

oltp_read_only options:
...
  --create_secondary[=on|off]   Create a secondary index in addition to the PRIMARY KEY [on]
...
```

On the line above that there is the link to https://raw.githubusercontent.com/pingcap/tidb-bench/master/sysbench/sysbench-patch/oltp_common.lua  I think it would be good to either update that file to be based on 1.0.20. Maybe upload versions for both 1.0.14 and 1.0.20?
Maybe we can advise to use `--create_secondary=off` and then later create the indexes?
Maybe we should contribute an improvement to sysbench, e.g. an `--create-indexes-at-the-end=on` ? We might have other things to add as well like support for `AUTO_RANDOM`?

Reply:
Looks like the `warmup` command was called `prewarm` in older versions.

Reply:
@dbsid what do you think about updating the `oltp_common.lua` file for 1.0.20?